### PR TITLE
AWS managed OTel layer Initial release

### PR DESCRIPTION
**Pull request by wangzlei**
1. support python3.8, nodejs, java, javaagent 4 knids
2. update sample applications
3. update README.md and CHANGELOG.me

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,35 @@
...
-
-As an event-driven, serverless computing platform, AWS Lambda runs user's code —knowns as **Lambda function**— in sandbox environment. So, launching OpenTelemetry Collector in Lambda environment needs the help of [AWS Lambda Extensions](https://aws.amazon.com/blogs/compute/introducing-aws-lambda-extensions-in-preview/). This project provides ADOT Lambda layers user can use directly. The layers embed both ADOT Collector(as a Lambda extension) and SDK, Lambda user can onboard OpenTelemetry with this solution out-of-the-box.
-
-[OpenTelemetry Python3.8 Lambda with sample](sample-apps/python-lambda/README.md) (Both SDK and Collector)
-
-[OpenTelemetry Lambda layer](extensions/aoc-extension//README.md) (Collector only)
-
+# AWS managed OpenTelemetry Lambda Layers
+
+As a downstream Repo of [opentelemetry-lambda](), ___aws-otel-lambda___ publishes AWS managed OpenTelemetry Lambda layers with preconfigure for use with AWS services and bundle reduced AWS Collector. Users can onboard OpenTelemetry in their existing Lambda functions by adding these ready-made layers directly.
```
Comment (willarmiros) [inline README.md:3]:
```suggestion
As a downstream Repo of [opentelemetry-lambda](https://github.com/open-telemetry/opentelemetry-lambda), ___aws-otel-lambda___ publishes AWS managed OpenTelemetry Lambda layers that are preconfigured for use with AWS services and bundle the reduced AWS Collector. Users can onboard to OpenTelemetry in their existing Lambda functions by adding these ready-made layers directly. 
```

File: README.md [hunk 2]
```diff
@@ -1,11 +1,35 @@
...
+    1. For SAM sample
+        ```
+        ./run.sh
+       ```
+       
+    2. For Terraform sample
+        ```
+       terraform init
+       terraform apply
+        ```
```
Comment (willarmiros) [inline README.md:31]: Include a `cd` command to be in correct directory?

**Review by willarmiros — APPROVED** [review]
Couple nits, lgtm!

File: README.md [hunk 3]
```diff
@@ -1,11 +1,35 @@
...
+
+As a downstream Repo of [opentelemetry-lambda](), ___aws-otel-lambda___ publishes AWS managed OpenTelemetry Lambda layers with preconfigure for use with AWS services and bundle reduced AWS Collector. Users can onboard OpenTelemetry in their existing Lambda functions by adding these ready-made layers directly. 
+- Python3.8 layer [aws-otel-python38-ver-1-1-0](https://aws-otel.github.io/docs/getting-started/lambda/python) contains OpenTelemetry Python v1.1.0 with Contrib v0.20b0
+- Nodejs layer [aws-otel-nodejs-ver-0-18-0](https://aws-otel.github.io/docs/getting-started/lambda/javascript) contains OpenTelemetry JavaScript v0.18.0 with Contrib v0.15.0
+- Java-Wrapper layer [aws-otel-java-wrapper-ver-1-1-0](https://aws-otel.github.io/docs/getting-started/lambda/java-manual-instr) contains OpenTelemetry Java v1.1.0
+- Java-Agent layer [aws-otel-java-agent-ver-1-1-0](https://aws-otel.github.io/docs/getting-started/lambda/java-auto-instr) contains AWS OpenTelemetry Java instrumentation v1.1.0
+
+
+
+## Sample
```
Comment (mxiamxia) [inline README.md:11]:
```suggestion
## Sample App
```

**Review by mxiamxia — APPROVED** [review]
Pls merge before addressing comments

**Comment by wangzlei** [comment]
Split README and CHANGELOG with comments fix in https://github.com/aws-observability/aws-otel-lambda/pull/75

**Pull request merged**